AirCasting/CreateSessionViews/ConfirmCreatingSessionView.swift

```diff
@@ -90,9 +90,9 @@ struct ConfirmCreatingSessionView: View {
                 .lineSpacing(9.0)
                 ZStack {
                     if sessionContext.sessionType == .mobile {
-                        GoogleMapView(pathPoints: [], isMyLocationEnabled: true, placePickerDismissed: Binding.constant(false), isUserInteracting: Binding.constant(true))
+                        GoogleMapView(pathPoints: [], isMyLocationEnabled: true, placePickerDismissed: Binding.constant(false), isUserInteracting: Binding.constant(true), mapNotes: .constant([]))
```

Comment:
we need to make a task for resolving this issue. I think splitting GoogleMapView into:
* `GoogleMapView` -> Plain, simple mapview with some customization points (used here and in other places that don't need measurements, notes etc. plotted)
* `AirCastingMapView` -> The map with features added used for displaying sessions